frontend/ui/uimanager.lua

```diff
@@ -1062,6 +1062,7 @@ function UIManager:_repaint()
             -- pass hint to widget that we got when setting widget dirty
             -- the widget can use this to decide which parts should be refreshed
             logger.dbg("painting widget:", widget.widget.name or widget.widget.id or tostring(widget))
+            Screen:forceRotation()
```

Comment:
I'd ideally prefer if we could (somehow ^^) ensure that either `forceRotation` or the actual `ioctl` only get called *once* per `_repaint`/stack, instead of before each widget.

Which is easier said than done ^^.

Reply:
Oh, you did, that's the `_forced_rotation` check, right?

(It's late, I haven't slept much,, I *really* shouldn't be trying to review this right now, sorry for the dumb comments ^^).